pkg/mutation/jsonpatch.go

```diff
@@ -0,0 +1,24 @@
+package mutation
+
+import (
+	"encoding/json"
+
+	"gomodules.xyz/jsonpatch/v2"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// GenerateJSONPatch generates a json patch from the original and mutated object
+func GenerateJSONPatch(original, mutated unstructured.Unstructured) ([]jsonpatch.JsonPatchOperation, error) {
```

Comment:
Should we use the library provided by controller-runtime for this?

https://github.com/kubernetes-sigs/controller-runtime/blob/d8385e3440270b807e132fc7fdb1cb769c1b6f06/pkg/webhook/admission/response.go#L83-L98

Reply:
Ah yes, did not know about that. Let me zap this, the conversion to json may be done in a helper function of the webhook